src/index.js

```diff
@@ -145,6 +143,24 @@ export default function auth(config = {}) {
     // Register error handling middleware for redirecting to support
     // redirecting on authentication failure.
     app.use(middleware.failedLogin(authOptions));
+
+    // Setup route handler for default success redirect
+    if (authOptions.shouldSetupSuccessRoute) {
```

Comment:
For web apps you are correct, you would pretty much always override them. However, if you use the pop-up method or are on mobile, with the defaults you don't need to do anything to the server. You check that you navigated to `/auth/success` grab the JWT from the cookie and close the pop-up window or the webview on mobile.